BDArmory/Control/MissileFire.cs

```diff
@@ -2080,6 +2080,58 @@ public int[] GetMissilesAway(TargetInfo target)
             engagedTargets = missilesAway.Count;
             //this.missilesAway = tempMissilesAway;
         }
+
+        public void CheckMissiles()
+        {
+            // Only do this check if we're in guard mode, assume that if a human player is control they're properly managing their radars
+            if (!guardMode) return;
+
+            MissileBase missileBase;
+
+            using (List<IBDWeapon>.Enumerator Missiles = BDATargetManager.FiredMissiles.GetEnumerator())
+                while (Missiles.MoveNext())
+                {
+                    if (Missiles.Current == null) continue;
+
+                    missileBase = Missiles.Current as MissileBase;
+
+                    if (missileBase.targetVessel == null) continue;
+                    // We assume that if the FiredByWM is the ParentWM that both vessels are on the same team, maybe not always a correct assumption?
```

Comment:
They will be on the same team, at least initially.
Something external could make them change teams later, but that could happen at any time with any pair of craft (e.g., while playing tag), so I assume that the missile logic either already accounts for that or ignores the possibility entirely.